psiturk/amt_services_wrapper.py

```diff
@@ -37,6 +37,7 @@
 from builtins import range
 from builtins import str
 from future import standard_library
```

Comment:
If we are only supporting > 3.5 then I think we only need `from __future__ import generator_stop` and `from __future__ import annotations`:


https://docs.python.org/3/library/__future__.html

feature | optional in | mandatory in | effect
nested_scopes | 2.1.0b1 | 2.2 | PEP 227: Statically Nested Scopes
generators | 2.2.0a1 | 2.3 | PEP 255: Simple Generators
division | 2.2.0a2 | 3.0 | PEP 238: Changing the Division Operator
absolute_import | 2.5.0a1 | 3.0 | PEP 328: Imports: Multi-Line and Absolute/Relative
with_statement | 2.5.0a1 | 2.6 | PEP 343: The “with” Statement
print_function | 2.6.0a2 | 3.0 | PEP 3105: Make print a function
unicode_literals | 2.6.0a2 | 3.0 | PEP 3112: Bytes literals in Python 3000
**generator_stop | 3.5.0b1 | 3.7 | PEP 479: StopIteration handling inside generators
annotations | 3.7.0b1 | 3.10 | PEP 563: Postponed evaluation of annotations**

Reply:
sounds good. the `__future__` stuff was left in there when support for python3 was added. And yeah we only support minimum python 3.6